tests/test_userstory_39.py

```diff
@@ -56,3 +62,87 @@ def test_melding_overlast():
         'object': verblijfsobject['url'],
     })
     assert 'url' in zaak_object
+
+
+    '''
+    Create temporary files to test file uploading
+    '''
+    # Text file (can't be empty or the API throws an error)
+    fd, txt_path = tempfile.mkstemp()
+    with open(txt_path, 'w') as tmp:
+        tmp.write('some additional information')
+
+    # Image (png format)
+    im_format = 'png'
+    im = Image.new('RGB', (512, 512), 'blue') #plain blue square
+    buffered = BytesIO()
+    im.save(buffered, format=im_format)
+
+    '''
+    Upload the files with POST /enkelvoudiginformatieobject (DRC)
+    '''
+
+    text_attachment = drc_client.create_form('enkelvoudiginformatieobject', {
+    'identificatie': uuid.uuid4().hex,
+    'bronorganisatie' : '1',
+    'creatiedatum' : zaak['registratiedatum'],
+    'titel' : 'detailed summary',
+    'auteur' : 'test_auteur',
+    'formaat' : 'txt',
+    'taal' : 'english',
+    }, {
+        'inhoud': open(txt_path, 'rb')
+    })
+
+    # Test if the EnkelvoudigInformatieObject stored has the right information
+    assert 'creatiedatum' in text_attachment
+    assert text_attachment['creatiedatum'] == zaak['registratiedatum']
+
+    # Retrieve the EnkelvoudigInformatieObject
+    txt_object_id = text_attachment['url'].rsplit('/')[-1]
+    text_attachment = drc_client.retrieve('enkelvoudiginformatieobject', id=txt_object_id)
+
+    # Test if the attached filed is our initial file
+    with open(txt_path, 'rb') as tmp:
+        assert urllib.request.urlopen(text_attachment['inhoud']).read() == tmp.read()
```

Comment:
`requests` library is more suited to do remote calls, let's forget about `urllib.request` alltogether :)

Reply:
got it